lib/winston-papertrail.js

```diff
@@ -136,6 +136,40 @@ var Papertrail = exports.Papertrail = function (options) {
     // Open the connection
     connectStream();
 
+    function cleanup() {
+        self._erroring = true;
+        try {
+            if (self.socket) {
+                self.socket.destroy();
+
+                //this throws some uncatchable thing, eww.
+                //self.socket.end();
+                self.socket = null;
+            }
+        }
+        catch (e) { }
```

Comment:
so the destroy here prevents the error from being thrown as opposed to close or end.  The try/catch does nothing to catch that socket error.